test/object_store/conftest.py

```diff
@@ -12,6 +12,7 @@
 sys.path.insert(1, sys.path[0] + '/../..')
 from test.pylib.minio_server import MinioServer
 from test.pylib.cql_repl import conftest
+from test.topology.conftest import *
```

Comment:
personally, i'd avoid the "import all" in practice. it's like the `using namespace some_other_namespace` in C++. but i don't insist.